src/components/docs/Docs/MobilePageSelect/MobilePageSelect.jsx

```diff
@@ -0,0 +1,25 @@
+import styles from './MobilePageSelect.module.scss';
+import { useNavigate, useLocation } from 'react-router-dom';
+
+export default function MobilePageSelect() {
+    const location = useLocation();
+    const currentPath = location.pathname;
+    const navigate = useNavigate();
+    const onSelectChange = event => {
+        const selectedPage = event.target.value;
+        if (selectedPage) {
+            navigate(selectedPage);
+        }
+    };
+    return (
+        <select className={styles.mobilePageSelect} onChange={onSelectChange} value={currentPath}>
+            <option value='/docs/'>Quickstart</option>
+            <option value='/app-registration/'>App Registration</option>
+            <option value='/api-explorer/'>API Explorer</option>
+            <option value='/docs/resources/api-guide/'>API Guide</option>
+            <option value='/docs/resources/faq/'>FAQ</option>
+            <option value='/docs/resources/json-schemas/'>JSON Schemas</option>
+            <option value='/docs/resources/bug-bounty/'>Bug Bounty</option>
```

Comment:
```suggestion
            const links = [
                { value: '/docs/', text: 'Quickstart' },
                { value: '/app-registration/', text: 'App Registration' },
                { value: '/api-explorer/', text: 'API Explorer' },
                { value: '/docs/resources/api-guide/', text: 'API Guide' },
                { value: '/docs/resources/faq/', text: 'FAQ' },
                { value: '/docs/resources/json-schemas/', text: 'JSON Schemas' },
                { value: '/docs/resources/bug-bounty/', text: 'Bug Bounty' },
            ]
            // and the following inside <select>:
            { links.map(({ value, text }) => <option key={value} value={value}>{text}</option> }
```
@hubert-deriv what do you think about mapping these options? 😊

Reply:
Ah the mobile page select! Well, that is strange, we removed the mobile page select actually, as it became redundant. Looks like it has been accidentally added again. I will remove it again.